src/docfx/docset/Docset.cs

```diff
@@ -163,6 +169,18 @@ private CultureInfo CreateCultureInfo(string locale)
             }
         }
 
+        private Dictionary<string, string> LoadResolveAlias(Config config)
+        {
+            var result = new Dictionary<string, string>(PathUtility.PathComparer);
+
+            foreach (var (alias, aliasPath) in config.ResolveAlias)
```

Comment:
`ResolveAlias` is used as a collection of tuples, not fetching value using a key. Does it have to be `Dictionary`?